test/bundle.c

```diff
@@ -172,7 +172,7 @@ static void bundle_test_resign(BundleFixture *fixture,
 	 * Note we have to use r_context() here as a hack to avoid re-setting
 	 * the context's 'pending' flag which would cause a re-initialization
 	 * of context and thus overwrite content of 'config' member. */
-	r_context()->config->keyring_path = g_strdup("test/openssl-ca/rel-ca.pem");
+	r_context_conf()->keyringpath = g_strdup("test/openssl-ca/rel-ca.pem");
```

Comment:
Why did you change r_context to r_context_conf? Doesn't the comment apply any more?